socketdev/purl/__init__.py

```diff
@@ -0,0 +1,36 @@
+import socketdev
+from urllib.parse import urlencode
+import json
+
+class Purl:
+    @staticmethod
+    def post(license="true", components=[]) -> dict:
+        path = "purl?" + "license="+license
+        components = {"components":components}
+        components = json.dumps(components)
+
+        response = socketdev.do_request(
+            path=path,
+            payload=components,
+            method="POST"
+        )
+        if response.status_code == 200:
+            purl = []
+            purl_dict = {}
+            result = response.text
+            result.strip('"')
+            result.strip()
+            for line in result.split("\n"):
+                if line != '"' and line != "" and line is not None:
+                    item = json.loads(line)
+                    purl.append(item)
+            for key, val in enumerate(purl):
```

Comment:
Any reason you're using `enumerate` instead of just getting things directly without?

like:

````
for item in purl:
    purl_id = item.get("id")
    purl_dict[purl_id] = item
````

Reply:
Lets change it then to keep the code more simple. Also it means that in a larger set there is less iteration going on.

Reply:
Updated not to use enumerate:
            for val in purl:
                purl_dict[val['id']] = val